profile/profile.md

```diff
@@ -0,0 +1,63 @@
+# Saiyeda Maham Shamail . Jarvis Consulting
+
+As a software developer I have been programming for the past 5 years, with one year in a professional setting. This has helped me develop exceptional problem solving skills. I have worked with technologies such as JavaScript, React.js, RDBMS (SQL), Redux and Python. I am passionate about learning new languages and technologies and I can do so very quickly.
+
+## Skills
+
+**Proficient:** JavaScript, RDBMS/SQL, Java, Android Development, Linux/Bash, Agile/Scrum, Git/GitHub/Gitflow, HTML5, XML, CSS3, JSON, Swift (IOS), MVC
+
+**Competent:** Python (Django), React.js and React Native, Redux, Google Firebase Cloud, JQuery, C/C++
+
+**Familiar:** Node.js, Jest, MongoDB (NoSQL), AWS S3, Docker
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_MahamShamail](https://github.com/jarviscanada/jarvis_data_eng_MahamShamail)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MahamShamail/tree/master/linux_sql)]: Developed a Linux Cluster Monitoring Agent (LCMA) software toolset. Designed LCMA to automate the collection, persistence and real-time monitoring of hardware specifications and resource usage data of all the nodes within a cluster of linux servers. Used CentOS 7 Google Cloud VM to develop and tests Bash scripts. Automated a bash script to create, start and stop a Docker contained PostgreSQL Server for data persistence. Implemented Bash CLI 'lscpu' and 'vmstd' commands to collect hardware specifications and resource usage data in two seperate Bash scripts. The resource usage data is automated to be collected in real time using 'crontab' script. Wrote SQL queries to perform data analysis, for example for identifying host failure, etc. Each feature was iteratively developed on GitHub on a seperate feature branch and later released after code reviews from a senior developer. All the scripts were tested to fail-fast on a single server, as well as manually. Followed best practicess of Gitflow, Scrum Framework and Agile through out the development, testing and deployment of this project.
```

Comment:
Developed a Linux Cluster Monitoring Agent (LCMA) software toolset using bash scripts, PostgreSQL, Docker, crontab, etc.

Reply:
it's not necessary to give too many details in your profile..it's more important to list all tech keywords (e.g. Linux, bash, Postgres, Docker, Git, etc..)

Developed a Linux Cluster Monitoring Agent (LCMA) software toolset that automates the collection, persistence and real-time monitoring of hardware specifications and resource usage data of a Google Cloud Linux cluster using bash scripts, PostgreSQL, Docker, crontab, etc. Wrote SQL queries to perform data analysis. Used Github/Gitflow to manage the source code.